### DISCUSSION TITLE
Any way to know which client created/modified note?

### DISCUSSION THREAD

**Discussion by meichthys** [TriliumNext/Trilium#3717] [Q&A] [answered]
I'm looking for a way to know which Trilium client identified a note. Ideally I'd like to name clients and expose (perhaps in the revision history) which client was the one that modified/created the note. I'm not familiar with the Trilium sync process, but it seems like the server should have a way to identify clients with some kind of UUID.

This would also help to know which client(user) was making revisions when there is more than one person editing notes (each with a different client).

**Comment by zadam** [comment, marked as answer]
There isn't really anything useful for that. The sync protocol has "instanceId", but it's ephemeral - it changes after each app restart.

**Reply by meichthys** [reply]
Ok, Thanks.
